examples/sample_dll.rs

```diff
@@ -5,14 +5,14 @@ use std::thread::spawn;
 use bindings::Windows::Win32::System::SystemServices::{
     DLL_PROCESS_ATTACH, DLL_PROCESS_DETACH, DLL_THREAD_ATTACH, DLL_THREAD_DETACH,
 };
-use mem::windows::wrappers::{DWORD, HMODULE, LPVOID};
+use mem::windows::wrappers::{HModule, DWORD, LPVOID};
 
 fn hack_thread() {
     println!("Hi from the sample dll");
 }
 
 #[no_mangle]
-pub extern "system" fn DllMain(module_handle: HMODULE, dw_reason: DWORD, lp_reserved: LPVOID) -> bool {
```

Comment:
Not sure whether or not we should have this variable named lp_reserved or reserved, I named it lp_reserved just incase someone who using the DLL macro doesn't name some other variable reserved and I wanted the DLL examples to be the same basically

Reply:
Must've been from rebasing at some point as I don't recall changing it. Reverted.